### ISSUE TITLE
Buffer time gets set to 200000 automatically

### ISSUE THREAD

**Issue by Bleuzen** [wwmm/pulseeffects#192]
When I start recording with an application (for example Audacity), the buffer-time always gets set to 200000, no matter what I set it to.

For example I set it to 50 in the GUI (50000).
Now when I start recording in Audacity, the effects do work, the GUI still displays the 50 ms buffer-time, but this is the output during this:
```
19:26:42.26 - PulseEffects - INFO - pipeline state: ready
19:26:42.29 - PulseEffects - INFO - pipeline state: playing
19:26:42.44 - PulseEffects - INFO - pulsesrc latency-time [us]: 10000
19:26:42.44 - PulseEffects - INFO - pulsesrc buffer-time [us]: 50000
19:26:42.45 - PulseEffects - INFO - pulsesink latency-time [us]: 10000
19:26:42.45 - PulseEffects - INFO - pulsesink buffer-time [us]: 50000
19:26:47.537 - PulseEffects - INFO - pipeline state: playing
19:26:47.539 - PulseEffects - INFO - pulsesrc latency-time [us]: 10000
19:26:47.539 - PulseEffects - INFO - pulsesrc buffer-time [us]: 200000
19:26:47.540 - PulseEffects - INFO - pulsesink latency-time [us]: 10000
19:26:47.540 - PulseEffects - INFO - pulsesink buffer-time [us]: 200000
19:26:47.587 - PulseEffects - INFO - pipeline state: ready
19:26:47.590 - PulseEffects - INFO - pipeline state: playing
19:26:47.593 - PulseEffects - INFO - pulsesrc latency-time [us]: 10000
19:26:47.593 - PulseEffects - INFO - pulsesrc buffer-time [us]: 200000
19:26:47.593 - PulseEffects - INFO - pulsesink latency-time [us]: 10000
19:26:47.593 - PulseEffects - INFO - pulsesink buffer-time [us]: 200000
19:26:50.929 - PulseEffects - INFO - pipeline state: ready
19:26:50.933 - PulseEffects - INFO - pipeline state: ready
```

So if I see it right, the buffer-time got upset to 200000, but I have it / it is still set to 50 in the GUI.
Is this correct, or a bug?

**Comment by wwmm** [comment]
The controls in the menu set latency and buffer time only for the pipeline processing apps audio output. The one processing the microphone uses GStreamer default values: 200 ms for the buffer and 10 ms for latency. 

I used to set these parameters for both pipelines but as you can see from some issues that were opened here the latency values are tricky to set. Depending on the values crashes may happen and it is not obvious which values are allowed in each hardware.  And to make things worse latency values that work for the apps output pipeline do not necessarily work for microphones. So eventually I decided to disable the application of these values to the mic pipeline.

In case you need to set these values for some reason I could add to the settings menu separate buffer and latency controls for each pipeline.

**Comment by Bleuzen** [comment]
Yes, I want to play with these values and tweak them as good as possible. So it would be nice, if you add options, to set it seperatly.

If you want, to avoid that to many users are opening issues here:
What about displaying a warning message like
"Hey, it is known that changing this values can cause issues."
the first time when a user tries to change them?

**Comment by Bleuzen** [comment]
@wwmm Can't test, see #199
